.drone.yml

```diff
@@ -3,19 +3,28 @@ kind: pipeline
 name: default
 
 steps:
+- name: download-dependencies
```

Comment:
@marwan-at-work out of curiousity, why download the dependencies in a separate step from the `go build`? Is it because you also run the `go mod vendor`?

Reply:
@arschles it's for security reasons, we don't want to expose the GCS credentials within the build step. Which leads me to another pull request that would only test GCP on merges (or maybe a nicer option where we only run _that_ test manually)

Reply:
ok, cool. makes sense!